qos-service-impl/build.gradle

```diff
@@ -0,0 +1,54 @@
+apply plugin: 'org.inferred.processors'
+apply plugin: 'org.unbroken-dome.test-sets'
+
+apply from: "../gradle/publish-jars.gradle"
+apply from: "../gradle/shared.gradle"
+apply from: "../gradle/timelock.gradle"
+
+testSets {
+    integTest
+}
+
+jacocoTestReport {
+    executionData test, integTest
+}
+
+check.dependsOn integTest
+dependencies {
+    compile (project(":qos-service-api"));
+
+    compile group: 'com.palantir.remoting3', name: 'tracing'
+    compile group: 'com.palantir.tritium', name: 'tritium-lib'
+    compile group: 'com.palantir.remoting3', name: 'jersey-servers'
+
+    processor group: 'org.immutables', name: 'value'
+
+    testCompile group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-yaml'
+    testCompile group: 'org.assertj', name: 'assertj-core'
+    testCompile(group: "org.jmock", name: "jmock", version: libVersions.jmock) {
+        exclude group: 'org.hamcrest'
+        exclude group: 'org.ow2.asm'
+    }
+    testCompile group: 'org.mockito', name: 'mockito-core'
+
+	integTestCompile (project(":atlasdb-config"));
+    integTestCompile group: 'io.dropwizard', name: 'dropwizard-core', version: libVersions.timelock_dropwizard
+    integTestCompile group: 'io.dropwizard', name: 'dropwizard-http2', version: libVersions.timelock_dropwizard
+    integTestCompile group: 'io.dropwizard', name: 'dropwizard-request-logging', version: libVersions.timelock_dropwizard
+    integTestCompile group: 'io.dropwizard.modules', name: 'dropwizard-java8'
+    integTestCompile group: 'io.dropwizard', name: 'dropwizard-testing', version: libVersions.timelock_dropwizard
+    integTestCompile group: 'com.palantir.remoting3', name: 'http2-agent'
+}
+
+configurations.all {
+     resolutionStrategy {
+         force 'io.dropwizard.metrics:metrics-core:3.2.5'
+         force 'org.hamcrest:hamcrest-core:1.3'
+         force 'org.slf4j:slf4j-api:1.7.21'
+         force 'org.hdrhistogram:HdrHistogram:2.1.10'
+         dependencySubstitution {
+             substitute module('io.dropwizard:dropwizard-jackson') with module('io.dropwizard:dropwizard-jackson:0.9.3')
+         }
+     }
+ }
```

Comment:
Would it make sense / be possible to restrict this to the `integTest` configuration?